src/client.ts

```diff
@@ -420,6 +422,22 @@ bad input into one of the iframe-coordinator client methods.
     });
   }
 
+  /**
+   * Asks the host application to open a modal dialog.
+   *
+   * This is identified by the ModalRequest's `modalId` property.
```

Comment:
The "This" in this sentence is ambiguous, IMO. I'd make it more explicit, something like:  "The `modalId` property names the modal that should be displayed"